bokehjs/src/coffee/renderer/guide/categorical_axis.coffee

```diff
@@ -26,6 +26,9 @@ class CategoricalAxis extends Axis.Model
 
     return range_bounds
 
+  display_defaults: () ->
+    return super()
+
```

Comment:
This also breaks `display_defaults`, because previously we used `extend()` on each subclass to collect all defaults, but now this will just get defaults for whatever `@type` is.